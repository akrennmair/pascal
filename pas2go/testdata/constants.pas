program test;

const a = 123;
	b = 3.1415;
	c = 'hello world!';

begin
	writeln('a = ', a);
	writeln('b = ', b);
	writeln('c = ', c)
end.
