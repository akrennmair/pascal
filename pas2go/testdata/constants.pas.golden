package main

import (
	"github.com/akrennmair/pascal/pas2go/system"
)

var _ = system.Write

// program test
func main() {
	const (
		a = 123
		b = 3.1415e0
		c = "hello world!"
	)

	system.Writeln("a = ", a)
	system.Writeln("b = ", b)
	system.Writeln("c = ", c)
}
